packages/react-next/src/components/ExtendedPicker/PeoplePicker/ExtendedPeoplePicker.doc.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { ExtendedPeoplePickerBasicExample } from '../examples/ExtendedPeoplePicker.Basic.Example';
+import { ExtendedPeoplePickerControlledExample } from '../examples/ExtendedPeoplePicker.Controlled.Example';
+
+import { IDocPageProps } from '../../../common/DocPage.types';
+
+const ExtendedPeoplePickerBasicExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/ExtendedPicker/examples/ExtendedPeoplePicker.Basic.Example.tsx') as string;
```

Comment:
These should be updated to react-next paths

Reply:
Oooh good catch!